app/assets/stylesheets/darkswarm/distributor_header.css.scss

```diff
@@ -11,7 +11,7 @@ section {
   display: block;
   background: $white;
   position: relative;
-  z-index: 2;
+  z-index: 20;
```

Comment:
Perhaps I'm not looking in the right place but it looks like this z-index property can just be removed? It doesn't seem to have any effect?
<img width="1438" alt="Screen Shot 2020-05-27 at 11 22 11 am" src="https://user-images.githubusercontent.com/2467577/82965759-6f904900-a00c-11ea-83cf-224356a001b4.png">

Reply:
The previous issue noted here: https://github.com/openfoodfoundation/openfoodnetwork/issues/5433#issuecomment-630089588 is broken again if we remove that line.